build.gradle

```diff
@@ -233,9 +234,11 @@ distributions {
 
 docker {
   javaApplication {
-    baseImage = 'openjdk:8-jre-slim'
+    // https://bmuschko.github.io/gradle-docker-plugin/current/user-guide/#extension_2
+    // update base image to openjdk 17 jdk slim. other jdk like termurin, etc generates error
```

Comment:
So does it work now, or not?

If the problem is with the image build, for instance if it is a program that is missing in the image, we could check if we can install it in the `dockerCreateDockerfile` task.

Reply:
> So does it work now, or not?

It does work. 
The comment is no longer valid. Sorry, I merged it so fast that saw your comment now and realised your confusion could have been from the comment. 
I will create a fixup PR to delete the comment.